frontend/src/pages/detail/components/TechStacksSection/TechStacksSection.tsx

```diff
@@ -0,0 +1,18 @@
+import IconBadgeList from "@shared/components/IconBadgeList/IconBadgeList";
+import SectionTitle from "../SectionTitle";
+import * as S from "./TechStacksSection.styled";
+
+interface TechStacksSectionProps {
+  techStacks: string[];
+}
```

Comment:
```suggestion
interface TechStacksSectionProps {
  techStacks: TechStack[];
}
```

> 저희 상수화를 다 맞춰놔서 string[] 보다는 플랫폼들에 대한 타입을 맞춰놔야 나중에 백엔드 api 받아와서 props으로 넣을 때도 타입 안전성이 더 좋을 것 같습니다~

여기도 `PlatformSection`컴포넌트와 동일합니다~